README.md

```diff
@@ -60,7 +64,7 @@ You can add issues and ask for help by putting an issue into this repository's
 issue queue.
 
 ## Contributing
-Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
+This is an open source project and encourage users to contribute. Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
 
 ## Tests
 This project comes with a simple testing tool to validate the playbook installs
```

Comment:
Add a period to the end of this sentence?

Reply:
I don't see the missing .